DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
 Any help to fix this is very welcome!
 
+#### Integration with VSCode and Delve
+
+If you would like to use the delve debugger for Go with VSCode to debug `gapis`, You can follow these steps
+
+1. Complete the fusing steps.
+
+2. Create a launch.json under the project directory `Ctrl + Shift + P` and `Debug: Open launch.json`
```

Comment:
"project directory" is ambiguous here, clarify that the user must save her vscode workspace under the <gapid-gofuse> directory, and then she can execute `Debug: Open launch.json`

Reply:
No, if you execute the command it will automatically create the directory and the file.